roles/ruby_s/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: ruby | check version
+  ansible.builtin.shell: ruby -v | awk '{print $2}'
+  register: installed_ruby
+
+- name: prints out the installed_ruby variables
+  ansible.builtin.debug:
+    msg: ruby version "{{ installed_ruby.stdout }}"
+
+- include_tasks: cleanup.yml
+  when: installed_ruby.stdout != "{{ desired_ruby_version }}"
```

Comment:
Per @carolyncole's comment today, we should update this condition to handle the case where no ruby is installed. Something like `when: installed_ruby.stdout != ""` in addition to the current condition.

Reply:
Or possibly `when: installed_ruby.stdout | length > 0`